internal/cli/cmd/login/login.go

```diff
@@ -224,9 +227,9 @@ func (o *option) newLoginOption() (loginOption, error) {
 
 func (o *option) selectAndSetProvider() error {
 	providers := o.remoteConfig.Security.Authentication.Providers
-	// The first step is to collect the different providers and store it into items + modifiers.
-	// items will be the selection items to display to users.
-	// modifiers will be the action to save the different user input into option struct.
+	// The first step is to collect the different providers and store it into items and modifiers.
+	// Items will be the selection items to display to users.
```

Comment:
Options now apparently

Reply:
I don't get it 😅

Reply:
The variable has been renamed from ``items`` to ``options``, but not the comments